mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -192,7 +192,7 @@ def runConfigWithMLIR(config):
     os.system("rm "+BENCHMARKING_RESULT_FILE_NAME)
     commandLineOptions = config.generateMlirDriverCommandLine()
     mlirMIOpenDriverCommand = os.path.join(MLIR_BUILD_DIR, MLIR_MIOPEN_DRIVER) + ' -ph -c ' + commandLineOptions
-    profilerCommand = ROCPROF + ' --hip-trace ' + os.path.join(MLIR_BUILD_DIR, MLIR_ROCM_RUNNER) + MLIR_ROCM_RUNNER_ARGS
+    profilerCommand = ROCPROF + ' --stats ' + os.path.join(MLIR_BUILD_DIR, MLIR_ROCM_RUNNER) + MLIR_ROCM_RUNNER_ARGS
```

Comment:
[Question] What's the difference between those two? Why are we changing this option now?

Reply:
The deps are things like boost, but the `make install` puts the MIOpen in the right place.

Reply:
--stats generates kernel execution statistics which is what we need here. --hip-trace generates more information. Both would work. I just think --stats is enough.